Netflix/SceneDelegate.swift

```diff
@@ -12,7 +12,10 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
 
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-        guard let _ = (scene as? UIWindowScene) else { return }
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        
+        self.window = UIWindow(windowScene: windowScene)
+        SplashCoordinator(window: self.window!).start()
```

Comment:
You should avoid using force unwrap almost always 
One of the exceptions can be IBOutlet, or when you need to inject something after your class was created, like in MVP pattern to create binding between view and presenter

In other cases, it's better to use `if let`, `if guard ` or `??`

```suggestion
        guard 
             let windowScene = (scene as? UIWindowScene),
             let window =  UIWindow(windowScene: windowScene)
         else { 
              return 
         }
        self.window = window
        SplashCoordinator(window: window).start()
```